go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/googleapis/gax-go/v2
```

Comment:
I could be wrong, but shouldn't this be only `gax-go` not `gax-go/v2`?

Reply:
I think aliases are definitely the way to Go. IIRC, we waiting to module-ize gax until Nov 1 exactly so that we could use aliases.

Reply:
I see that I misread https://github.com/golang/go/wiki/Modules#releasing-modules-v2-or-higher. Instead of the "Major branch" version, we're doing a combination of "Major branch" and "Major subdirectory". Updated to include a go.mod in the subdirectory, and made the parent v1.